src/app/descriptors/income/income8dig/text-template.html

```diff
@@ -0,0 +1,74 @@
+בשנת {{year}} הכנסת {{title}}
+{% if net_allocated %}
+ עמדה על {{net_allocated | format_number}}₪.
+{% endif %}
+{% if not net_allocated %}
+  לא הוקצה.
+{% endif %}
+
+{% if (net_allocated != net_revised) %}
+<br />
+אחרי העברות תקציביות, התקציב עמד על {{net_revised | format_number}}₪
```

Comment:
not sure this applies here (I think there are no העברות for income items, but please make sure)